### PR TITLE
Redirect support for old version routes 

### PR DESCRIPTION
**Pull request by siva-kannan3**
#### Changes

- Added redirect for v1 ```/editor``` route to ```/builder```
- Added disabled cursor for disabled template navigation arrows
- Added total templates count in top nav bar

Fixes # (issue)

#### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

#### Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx [hunk 1]
```diff
@@ -48,9 +48,11 @@ export const TemplateSlider = () => {
           },
           '.splide__arrow--prev': {
             backgroundColor: 'transparent',
+            display: 'none !important', // #TODO: revert if more than templates available
```
Comment (gopal1996) [inline src/modules/builder/nav-bar/components/TemplatesSlider.tsx:51]: if no template are there further, is there any indicator? if present, we can hide arrow using that

File: next.config.js [hunk 2]
```diff
@@ -3,18 +3,27 @@ const nextConfig = {
...
+      'src/helpers',
+      'src/modules',
+      'src/pages',
+      'src/styles',
+      'src/templates',
     ],
   },
   images: {
-    domains: ["avatars.githubusercontent.com"],
+    domains: ['avatars.githubusercontent.com'],
```
Comment (gopal1996) [inline next.config.js:17]: are we using this domain anywhere?

Comment (gopal1996) [reply next.config.js:17]: if not can we remove this?

File: src/modules/builder/nav-bar/components/TemplatesSlider.tsx (hunk 1 above)
Comment (siva-kannan3) [reply src/modules/builder/nav-bar/components/TemplatesSlider.tsx:51]: No, we don't have any indicator for no templates

Comment (siva-kannan3) [reply src/modules/builder/nav-bar/components/TemplatesSlider.tsx]: For now, planned to remove the hide logic, as there will be templates introduced in near future, then the issue won't even exist. and we also introduced cursor as disabled now so even if user is going to click the disabled arrow, they immediately get the feedback from ux perspective.

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**